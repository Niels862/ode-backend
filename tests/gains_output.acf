chip {
    io: [
        output,
        output,
        output,
        output,
    ],
    cabs: [
        cab 1 with clocks 1, - {
            cams: [
                GainInv as gain1 {},
                GainInv as gain2 {}
            ]
        }
    ],
    routing: [
        gain1 -> io1,
        gain1 -> io3,
        gain2 -> io2,
        gain2 -> io4,
    ]
}