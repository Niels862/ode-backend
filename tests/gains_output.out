0 0 0 0 32 4 0 2 5 0 0 64 0 0 81 255 15 241 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 64 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 2 255 0 0 0 86 0 0 0 200 0 0 16 0 0 16 0 0 16 0 0 16 0 0 0 0 0 0 0 0 0 0 0 0 255 255 255 255 255 255 255 255 0 0 0 0 0 0 0 0 1 18 1 130 1 16 1 130 1 19 1 129 1 16 1 129 12 0 17 0 0 0 0 0 0 0 0 0 0 32 0 32 0 0 0 32 0 5 0 48 0 16 0 5 0 0 0 16 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0